### PR TITLE
Milestone1: Create a new database, create the animals table schema, insert data into it and query the table to obtain required data.

### PR DESCRIPTION
**Pull request by carlylechia**
To complete this milestone,

- [x] I created the `vet-clinic-database`
- [x] I created the `animals table`
- [x] I populated the table with the sample data provided in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md)
- [x] I wrote queries for all the data requested for in the project requirements.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (sja-thedude) [inline queries.sql]: - [x] Kindly replace the 2nd query with this as it should be written in this way. 
`SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+This repository includes files with plain SQL that can be used to recreate a database:
+
+- [schema.sql](./schema.sql) contains all tables.
+- [data.sql](./data.sql) contains sample data used to populate the tables.
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+
+
+## Authors
+
+👤 **Author1**
```
Comment (sja-thedude) [inline README.md]: - [x] Kindly replace Author 1 with your name.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @carlylechia, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job so far
✔️ Correct gitflow
✔️ Good PR title and description

## Required 
- [ ] Kindly make sure to add screenshots of your queries results to the pull request description.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi @carlylechia,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by carlylechia** [comment]
> Hi @carlylechia, Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> ## Highlights
> * The Pull Request has a proper title and description. ✔️
> * You have added a descriptive Readme file. Good Job. ✔️
> * There is no linter error is present. ✔️
> * Project has met all the technical aspects you needed for this project. Keep rocking! ✔️
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Please can you help verify why my dashboard still says: `review pending` ? I am unable to submit as a result of that.

**Pull request merged**